### ISSUE TITLE
feat: env variable support for drivers

### ISSUE THREAD

**Issue by tulilirockz** [ublue-os/uupd#22]
Would be nice to have that so that we could do integration testing easier and we could run uupd directly from a container by mouting something like `/run/host`

**Comment by tulilirockz** [comment]
This is a rather out-of-scope thing that would require namespacing support in this. I believe just making it so there are a few environment variables that control where the binaries are is the best implementation we could have for this

**Comment by tulilirockz** [comment]
Requires: #30
